l2geth/internal/ethapi/api.go

```diff
@@ -1041,6 +1041,9 @@ func DoEstimateGas(ctx context.Context, b Backend, args CallArgs, blockNrOrHash
 
 	// 3. calculate the fee and normalize by the default gas price
 	fee := core.CalculateRollupFee(*args.Data, uint64(gasUsed), dataPrice, executionPrice).Uint64() / defaultGasPrice
+	if fee < 21000 {
+		fee = 21000
+	}
```

Comment:
Note that we probably should set the default gasPrice to `0.1 gwei` considering our minimum gas limit has to be 21k. At 21k the minimum fee would end up being `0.000021 ETH` which is just a little too high

Reply:
Agree with this, just a matter of changing the value of `defaultGasPrice`

Reply:
I've updated the `defaultGasPrice` to be `0.1` gwei